src/approvals-service.ts

```diff
@@ -31,6 +32,7 @@ export async function getFormSubmissionApprovals(
     Sentry.captureException(error)
     console.error('Error retrieving form submission approvals', error)
 
+    if (!(error instanceof HTTPError)) throw error
```

Comment:
This will dramatically change the logic below. Will need to rethink this.

Reply:
How so? Everything below relies on it being an `HTTPError`

Reply:
The next line does not. And line 74 is what would have happened if `!(error instanceof HTTPError)` That is quite different to this.